rebuild/lib/index.js

```diff
@@ -5,6 +5,30 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 const bindings = require("bindings")("blst_ts_addon");
 
+bindings.verify = function verify(msg, pk, sig) {
+  return bindings.aggregateVerify([msg], [pk], sig);
+};
+
+bindings.asyncVerify = function asyncVerify(msg, pk, sig) {
+  return bindings.asyncAggregateVerify([msg], [pk], sig);
+};
+
+bindings.fastAggregateVerify = function fastAggregateVerify(msg, pks, sig) {
+  try {
+    return bindings.aggregateVerify([msg], [bindings.aggregatePublicKeys(pks)], sig);
```

Comment:
is it documented anywhere that `fast...` versions don't throw and the non fast ones do?

Reply:
I didn't know that was the case actually.  I will write some docstrings for that!!

Reply:
Updated.

Reply:
I kinda forgot why I had those there until I turned the tests back on.  If an invalid public key is passed the aggregatePublicKeys throws not returns false per the spec.  The actual verification will return false with a zero key but its not possible to create an invalid key similar to what we saw in the first PR unit tests.